frontend/app/views/event/page.scala.html

```diff
@@ -84,19 +84,29 @@ <h1 class="event-masthead__name">@event.name.text</h1>
     @if(!event.isBookable) {
         <div class="event-header event-header--unavailable">
             <div class="event-header__container event-header__container--details event-header__container--unavailable">
-                <div class="event-ticket__details">
-                    <span class="block-level event-ticket__details__status">
-                        @if(event.isSoldOut && event.status == "live") { Sold out } else { Past event }
-                    </span>
-                    <span class="block-level event-ticket__details__blurb">Tickets are no longer available</span>
+                <div class="event-ticket event-ticket--aligned">
+                    <div class="event-ticket__details">
+                        <span class="block-level event-ticket__details__status">
+                            @if(event.isSoldOut && event.status == "live") { Sold out } else { Past event }
+                        </span>
+                    </div>
+                    <div class="event-ticket__action">
+                        @if(event.isSoldOut && event.status == "live") {
```

Comment:
This whole check is really messy, but I'm hoping I'll be able to do something nice once https://github.com/guardian/membership-frontend/pull/66 is in.